chain/cosmos/ics.go

```diff
@@ -55,23 +57,27 @@ func (c *CosmosChain) FinishICSProviderSetup(ctx context.Context, r ibc.Relayer,
 
 	providerVal := stakingVals[0]
 
-	beforeDel, err := c.StakingQueryDelegationsTo(ctx, providerVal.OperatorAddress)
```

Comment:
i don't really get what happened here, but as soon as I added a second validator to the ICS test, StakingQueryDelegationsTo started returning delegation responses that only had the second delegation and lacked the first self-delegation. So the balance after was less than the balance before; it seems if you just query the validators and get the balance from there it works out, so that's what I did.